src/domain/fiat/index.types.d.ts

```diff
@@ -2,7 +2,8 @@ type UsdCents = number & { readonly brand: unique symbol }
 type CentsPerSatsRatio = number & { readonly brand: unique symbol }
 type DisplayCurrencyBaseAmount = number & { readonly brand: unique symbol }
 type DisplayCurrency =
-  typeof import(".").DisplayCurrency[keyof typeof import(".").DisplayCurrency]
+  | typeof import(".").DisplayCurrency[keyof typeof import(".").DisplayCurrency]
+  | (string & { readonly brand: unique symbol })
```

Comment:
Is this to support currencies that we don't specifically encode in the `DisplayCurrency` object?

Reply:
Just leaving a note after offline chat: the supported currencies come from the price service so we're thinking that having an enum here potentially doesn't make sense since it's not the source of truth.

There are lots of places in the code as well that use the `DisplayCurrency.Usd` value.

We can probably revisit in another PR to see if that usage makes sense or if we do something like create a simple constant and make the `DisplayCurrency` type just a unique string.